forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/FormPDFListener.java

```diff
@@ -0,0 +1,319 @@
+package org.camunda.bpm.extension.hooks.listeners;
+
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.Playwright;
+import com.microsoft.playwright.options.Margin;
+import com.microsoft.playwright.options.Media;
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateExceptionHandler;
+import org.apache.commons.lang.StringUtils;
+import org.camunda.bpm.engine.delegate.DelegateExecution;
+import org.camunda.bpm.engine.delegate.DelegateTask;
+import org.camunda.bpm.engine.delegate.Expression;
+import org.camunda.bpm.engine.delegate.TaskListener;
+import org.camunda.bpm.extension.commons.connector.HTTPServiceInvoker;
+import org.camunda.bpm.extension.hooks.exceptions.FormioServiceException;
+import org.camunda.bpm.extension.hooks.services.FormSubmissionService;
+import org.camunda.bpm.extension.hooks.services.IMessageEvent;
+import org.camunda.bpm.extension.mail.EmptyResponse;
+import org.camunda.bpm.extension.mail.config.MailConfiguration;
+import org.camunda.bpm.extension.mail.config.MailConfigurationFactory;
+import org.camunda.bpm.extension.mail.send.SendMailInvocation;
+import org.camunda.bpm.extension.mail.send.SendMailRequest;
+import org.camunda.bpm.extension.mail.service.MailService;
+import org.camunda.bpm.extension.mail.service.MailServiceFactory;
+import org.camunda.connect.impl.AbstractConnector;
+import org.camunda.connect.spi.ConnectorRequestInterceptor;
+import org.camunda.connect.spi.ConnectorResponse;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import javax.activation.DataHandler;
+import javax.activation.DataSource;
+import javax.activation.FileDataSource;
+import javax.inject.Named;
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import javax.ws.rs.core.Application;
+import java.io.*;
+import java.nio.file.Paths;
+import java.util.*;
+
+@Named("FormPDFListener")
+public class FormPDFListener extends BaseListener implements TaskListener, IMessageEvent {
+
+    protected MailConfiguration configuration;
+
+    private String formUrl;
+    private String managerEmail;
+    private String submitterEmail;
+
+    @Autowired
+    private FormSubmissionService formSubmissionService;
+
+    @Autowired
+    private HTTPServiceInvoker httpServiceInvoker;
+
+    @Override
+    public void notify(DelegateTask delegateTask) {
```

Comment:
I'm wondering if we maybe could combine this listener with the `TaskAssignmentListener` to make it a bit more flexible and the PDF to be included in any email you desire. 

E.g. allow passing along a `attachSubmission` field that determines whether or not the PDF should be attached.

![image](https://user-images.githubusercontent.com/66635118/157558770-fb359006-50be-446e-a94b-63ef04d568b0.png)